### ISSUE TITLE
Propose new Layout cop to enforce assignment style

### ISSUE THREAD

**Issue by rpbaptist** [rubocop-hq/rubocop#5673]
Sometimes the method calls after an assignment are longer than 80 characters. I see different styles occurring and I would like to propose a cop, disabled by default, which can enforce this rule if enabled.

It is similar to [Layout/MultilineAssignmentLayout](https://github.com/bbatsov/rubocop/blob/master/manual/cops_layout.md#layoutmultilineassignmentlayout) except it would not just be for multiline assignments.

```ruby
# good
foo = expression.method_name(a: a, b: b)

# good
foo = expression.method_name(
  a: a, b: b
)

# good
foo = expression.method_name(
  a: a, 
  b: b
)

# good
foo = expression.method_name(a: a,
                             b: b)

# bad
foo = 
  expression.method_name(a: a, b: b)

```

As you see it would basically check if there is a new line or not after an assignment.

I'd be happy to give this a go, but I am making this issue to check if it is desirable.
